finality-aleph/src/sync/mock/backend.rs

```diff
@@ -100,9 +101,14 @@ impl Backend {
         Self {
             inner: storage,
             notification_sender,
+            headers_are_correct: Arc::new(Mutex::new(true)),
         }
     }
 
+    pub fn start_discarding_headers(&mut self) {
+        *self.headers_are_correct.lock() = false;
```

Comment:
any way to switch it back into accepting headers?

Reply:
currently this is not required by any test